subset/pentests/brute_client.py

```diff
@@ -0,0 +1,79 @@
+import socket, sys, json
+
+user_data=open('user_list.json')
+user_list = json.load(user_data)
+user_data.close()
+
+#username=['root','admin','manager']
+username= user_list['username'].split(",")
+
+#password=['root','pass','friend']
+password= user_list['password'].split(",")
+
+arguments = sys.argv
+
+print 'brute_client test begin'
+print 'address:' + arguments[1]
+print 'port:' + arguments[2]
+print 'reportFile:' + arguments[3]
+
+#address = '127.0.0.1'
+address = str(arguments[1])
+
+#port = 10000
+port = int(arguments[2])
+
+report_file = str(arguments[3])
+data_length =64
+
+expected=['login:','Password:','Last login:','Login incorrect','Connection closed by foreign host.']
+expected1=['username:','password:','success','login failed','Connection closed by foreign host.']
+
+sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+server_address = (address,port)
+
+sock.connect(server_address)
+
+login_report='\n'
+u=0
```

Comment:
avoid super short variable names. I have no idea what "u" means... no need to conserve memory!

Reply:
extended naming convention to user_count